### PR TITLE
Add missing custom-* data (Part 2)

### PR DESCRIPTION
**Pull request by ApCoder123**
Part of #6845

### CODE REVIEW

File: entries/h/harvard.edu.json [hunk 1]
```diff
@@ -8,6 +8,12 @@
       "custom-software",
       "custom-hardware"
     ],
+    "custom-software": [
+      "Duo"
+    ],
+    "custom-hardware": [
+      "Yubico OTP"
```
Comment (Carlgo11) [inline entries/h/harvard.edu.json:15]: I assume "HarvardKey" are Yubikeys but I can't find any documentation that the OTP feature is the method being used.

File: entries/i/invoiced.com.json [hunk 2]
```diff
@@ -5,7 +5,10 @@
       "sms",
       "custom-software"
     ],
-    "documentation": "https://support.invoiced.com/knowledge_base/topics/setting-up-two-factor-authentication",
+    "custom-software": [
+      "Authy"
+    ],
+    "documentation": "https://docs.invoiced.com/faqs#wb-does-invoiced-support-two-factor-authentication-2fa",
```
Comment (Carlgo11) [inline entries/i/invoiced.com.json]: The other link doesn't direct me to the right part of the page.
```suggestion
    "documentation": "https://docs.invoiced.com/faqs#wBTZz",
```

File: entries/w/wellsfargo.com.json [hunk 3]
```diff
@@ -9,7 +9,14 @@
     "tfa": [
```
Comment (Carlgo11) [inline entries/w/wellsfargo.com.json:9]: Not sure if they provide 2FA.

>To protect your personal information and guard against unauthorized access to your accounts, we may require a code when you:
>* Access certain Wells Fargo accounts or services.
> * Send money to another person that you haven’t transferred money to before, or receive money from a non-Wells Fargo account that you haven’t used before.

Comment (phallobst) [reply entries/w/wellsfargo.com.json:9]: According to https://www.wellsfargo.com/privacy-security/fraud/articles/two-factor-authentication/, they do:
> Customers can also add an additional layer of security by activating Wells Fargo's 2FA feature, 2-Step Verification at Sign-On.

File: entries/h/harvard.edu.json (hunk 1 above)
Comment (ApCoder123) [reply entries/h/harvard.edu.json:15]: There was a reference to "Yubikey passcode" on [this page](https://harvard.service-now.com/ithelp?id=kb_article&sys_id=97c56d093778f68036fdd8d543990e78) and I assumed that was referring to OTP

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**